docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,199 @@
+Enable cross-cluster replication in Aiven for Apache Cassandra® |beta|
+======================================================================
+
+Enabling the cross-cluster replication (CCR) feature requires building a CCR setup in the Aiven platform and, next, configuring the replication on the Apache Cassandra side. This article covers the first part only by providing instructions on how to set up CCR for your Aiven for Apache Cassandra® service on the Aiven side.
+
+.. For the other part (the configuration of the replication factor on the Apache Cassandra side), which ultimately makes the replication work, see the instruction in **Set up the replication factor**.
+
+About enabling CCR
+------------------
+
+You can enable CCR either when creating a new Aiven for Apache Cassandra service or on an existing service.
+
+.. note::
+
+   Enabling CCR on an existing service is only possible if the service has ``NetworkTopologyStrategy`` set as a replication strategy for the keyspace.
```

Comment:
> for the keyspace

`for all distributed keyspaces` would be better, since the user can have more than one keyspace